.github/workflows/cla.yml

```diff
@@ -0,0 +1,25 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          # You do not need to create this file manually.
+          path-to-signatures: 'cla/signatures/version-1/cla.json'
+          path-to-document: 'https://github.com/datafuselabs/datafuse/blob/master/cla/agreements/DCLA.md'
```

Comment:
path to `website/datafuse/docs/cla/agreements/DCLA.md`?

Reply:
> path to `website/datafuse/docs/cla/agreements/DCLA.md`?

Why not put everything related to cla together? We can use soft links to provide a copy for docs.

Reply:
ok +1

Reply:
BTW, Clickhouse approach is very simple:

https://github.com/ClickHouse/ClickHouse/blob/master/.github/PULL_REQUEST_TEMPLATE.md

His robot will check the PR contains:
```
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
```
By retaining this you are agreeing to their CLA and do not need to sign.

Reply:
![image](https://user-images.githubusercontent.com/172204/120751523-b9264480-c53a-11eb-83b6-11d2ca40a2ce.png)
But the complex is to have status check, I think this patch is to do it.

Reply:
For us, just add a comment like this

```
I have read the CLA Document and I hereby sign the CLA
```

Reply:
For this PR, the expected behavior is just like here: https://github.com/nix-dot-dev/nix.dev/pull/40